scripts/extract_keys.py

```diff
@@ -0,0 +1,39 @@
+import sys
```

Comment:
@snwiem - could you remove this script? This repository is for translations not translation tools :)